src/rpc/common/Types.h

```diff
@@ -53,4 +53,14 @@ struct FieldSpec;
 
 using RpcSpecConstRef = RpcSpec const&;
 
+struct VoidOutput
+{
+};
+
+inline void
+tag_invoke(boost::json::value_from_tag, boost::json::value& jv, VoidOutput o)
```

Comment:
No need for a name for the VoidOutput parameter. This would have been a warning turned error when we introduce flags again.. i'm sure it would work without the name :)